libs/designsystem/src/lib/components/chart/options/activitygauge.ts

```diff
@@ -82,7 +82,7 @@ export const ActivityGaugeOptions: Options = {
   series: [
     {
       type: 'solidgauge',
-      data: this === undefined ? [] : this.data,
+      data: [],
```

Comment:
This doesn't seem like a safe refactoring... 🔥 
See this PR for an alternative solution: https://github.com/kirbydesign/designsystem/pull/1260

Reply:
Thank you, @jakobe. I was uncomfortable doing that. In the Chart component it looks like it always get overridden, though. But I prefer the solution you mention.